src/library/styles/abstracts/_vars-mixins.scss

```diff
@@ -0,0 +1,40 @@
+/* stylelint-disable declaration-empty-line-before */
+@use 'sass:map';
+
+// Gradient animation keyframes
+@keyframes animated-gradient {
+  0% {
+    background-position: 100% 0;
+  }
+
+  100% {
+    background-position: 0 0;
+  }
+}
+
+@mixin linear-gradient($direction, $style) {
+  background: linear-gradient($direction, $style);
+}
+
+@mixin border-gradient($direction, $style) {
+  border-image-source: linear-gradient($direction, $style);
+}
+
+@mixin animated-gradient($style) {
+  $animatedgradient: map.get($animatedgradients, $style);
+
+  background: $animatedgradient;
+  background-image: linear-gradient($vertical, $animatedgradient);
+  background-size: 400% 100%;
+  animation: animated-gradient 10s linear infinite;
+
+  @if $style != $brand {
+    background-size: 300% 100%;
+  }
```

Comment:
since this is the standard, maybe consider flipping so 300% 100% is default and if brand = 400